ractor_example_entry_proc/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
```

Comment:
does this need to be published to crates.io? or is it only for tests?

We might need to update our publish CI pipeline.